packages/vite-plugin-shopify/README.md

```diff
@@ -36,12 +36,12 @@ export default {
     /* Plugin options are not required, defaults shown */
     viteShopify({
       // Root path to your Shopify theme directory (location of snippets, sections, templates, etc.)
-      themeRoot: "./",
-      // Front-end source code directory
+      themeRoot: process.cwd(),
+      // Front-end source code directory (relative to themeRoot)
       sourceCodeDir: "frontend",
-      // Front-end entry points directory
-      entrypointsDir: "frontend/entrypoints",
-      // Additional files to use as entry points (accepts an array of file paths or glob patterns)
+      // Front-end entry points directory (relative to sourceCodeDir)
+      entrypointsDir: "entrypoints",
```

Comment:
See my comment above about why I didn't want to have these options be relative to each other (ensuring maximum flexibility for developers).